src/pages/login.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { ReactElement } from 'react';
+import { NextPageContext } from 'next';
+import { parseCookies } from 'nookies';
+import Layout from '../layout/Layout';
+import ErrorSummary from '../components/ErrorSummary';
+import FormElementWrapper from '../components/FormElementWrapper';
+import { OPERATOR_COOKIE } from '../constants';
+import { ErrorInfo } from '../types';
+import { deleteCookieOnServerSide } from '../utils/index';
+
+const title = 'Login - Fares data build tool';
+const description = 'Login page of the Fares data build tool';
+
+interface LoginProps {
+    errors: ErrorInfo[];
+}
+
+const Login = ({ errors = [] }: LoginProps): ReactElement => {
+    return (
+        <Layout title={title} description={description}>
+            <main className="govuk-main-wrapper app-main-class" id="main-content" role="main">
+                <div className="govuk-grid-row">
+                    <div className="govuk-grid-column-two-thirds">
```

Comment:
I think the page will inherit 2 thirds columns from the layout above, meaning here we are only using 2 thirds of the page in its entirety. I think Laurence made a few different Layouts that can be used.

Reply:
Now replaced with new component - BaseLayout.